### ISSUE TITLE
recordlinkage.preprocessing module is missing

### ISSUE THREAD

**Issue by gwerbin** [J535D165/recordlinkage#48]
I got this error after installing with Pip from the PyPI wheel, PyPI source, and GitHub source:

```python
import recordlinkage.preprocessing
```
```
ModuleNotFoundError: No module named 'recordlinkage.preprocessing'
```

So I looked and found that, in all three cases, the `preprocessing` module was missing:

```shell
cd <stuff>/site-packages
ls -1XF recordlinkage
```
```
algorithms/
datasets/
__pycache__/
standardise/
base.py
classifiers.py
comparing.py
indexing.py
__init__.py
measures.py
rl_logging.py
types.py
utils.py
_version.py
```

And it looks like [preprocessing is missing from `setup.py`](https://github.com/J535D165/recordlinkage/blob/master/setup.py#L36-L39). Is this intentional? What's the correct usage of `recordlinkage.preprocessing`? The documentation uses that module by name in the examples.

**Comment by J535D165** [comment]
Good catch! Stupid mistake :(

Checkout commit 18b9634256b0173852ec5ad21ad286c92822179b. Will release 0.11.1 release today.

I will open a new issue to improve the CI part. CI didn't fail...

**Comment by J535D165** [comment]
Released 0.11.1 and 0.11.2. 

0.11.2 excludes the benchmarks.
